File: python/nivel 2 logica/14.py
"""
Ejercicio 14
Eliminar duplicados de una lista
Sin usar set()
"""
numeros = [4,15,2,4,89,15,23,2,7]

sin_duplicados = []

for numero in numeros:
  if numero not in sin_duplicados:
    sin_duplicados.append(numero)

print(f"Lista sin duplicados {sin_duplicados}")

File: javascript/nivel 2 logica/14.js
/*
Ejercicio 14
Eliminar duplicados de una lista
Sin usar Set
*/

let numeros = [4,15,2,4,89,15,23,2,7];

let sinDuplicados = [];

for (let i = 0; i < numeros.length; i++) {
  if(!sinDuplicados.includes(numeros[i])){
    sinDuplicados.push(numeros[i]);
  }
}

console.log("lista sin duplicados", sinDuplicados);